lib/MessageDecoder.ts

```diff
@@ -87,6 +88,14 @@ export class MessageDecoder {
       }
     }
 
+    // C-Band puts a 10 char header in front of some message types
+    // First 4 chars are some kind of message number
+    // Last 6 chars are the flight number
+    let cband = message.text.match(/^(?<msgno>[A-Z]\d{2}[A-Z])(?<airline>[A-Z]{2})(?<number>[0-9]{4})/);
```

Comment:
For `airline` capture group, we might have to be a bit more forgiving in terms of what is allowed since the following are valid IATA callsigns: `EK05RQ`, `B60001`, `9E0001`, `MCEY42`

Reply:
seems like there's a possible full format described here: https://en.wikipedia.org/wiki/Airline_codes#IATA_airline_designator

It makes sense to let the airline be letters or digits, but the C-Band prefix is a fixed 10 characters as far as I've seen, and it pads out the numeric part to 4 digits, so there's no room for the third airline designator character or optional operational suffix. I could change it anyway if you like but I think it makes sense to try and keep it as tight as possible to try to avoid false matches.